test/util-test.js

```diff
@@ -75,9 +76,34 @@ test('urlTemplate() returns different templates based on pkg and rc', function (
   }
   var t6 = util.urlTemplate(o6)
   t.equal(t6, 'http://foo.com/w00t/{name}-{major}.{minor}-{runtime}-v{abi}-{platform}-{arch}.tar.gz', 'pkg.binary.package_name is added after host and remote_path, custom format')
-  var o7 = {pkg: require('../package.json'), download: true}
+  var o7 = {
+    pkg: xtend({}, require('../package.json')),
```

Comment:
The first argument `{}` is not necessary

Reply:
true, it's different from `Object.assign` in that respect

Reply:
@vweevers Removed in commit after merge.